src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -157,9 +157,9 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('iot central app', iot_central_sdk, client_factory=iot_central_service_factory,
                             is_preview=True) as g:
-        g.custom_command('create', 'iot_central_app_create')
+        g.custom_command('create', 'iot_central_app_create', supports_no_wait=True)
         g.custom_command('list', 'iot_central_app_list')
         g.custom_show_command('show', 'iot_central_app_get')
         g.generic_update_command('update', getter_name='iot_central_app_get',
                                  setter_name='iot_central_app_update', command_type=update_custom_util)
-        g.custom_command('delete', 'iot_central_app_delete', confirmation=True)
+        g.custom_command('delete', 'iot_central_app_delete', confirmation=True, supports_no_wait=True)
```

Comment:
It's better to add `g.wait_command`

Reply:
I saw most usage of g.wait_command is g.wait_command("wait"), it seems it has separate purpose from supports_no_wait, what's it for?

Reply:
If users use `--no-wait`, they need a way to looply query the latest status and return when the provisioningState is Success. `wait` command works for such scenario. @kawayiboy If this PR is urgent, you can ignore this comment since the severity is medium.

Reply:
@MyronFanQiu , I added  g.wait_command("wait"), let me know if it works